### ISSUE TITLE
Add support for vgy.me

### ISSUE THREAD

**Issue by TechKat** [ShareX/ShareX#549]
I own a little micro image hosting service named vgy.me and I'd like to get its name out there and thought ShareX could help.

At the moment, there is no official API, however I am allowing users to make POST requests to http://vgy.me/upload which will always return a JSON string containing necessary objects.

If any improvements or suggestions need to be made to vgy.me before it is considered, please let me know and I will do what I can to meet your standards.

Thanks.

**Comment by Jaex** [comment]
Normally I don't add not known uploaders because mainly those close service like in one year and then my efforts going waste while implementing them. But still I liked simplicity of your site and will add it.

By the way when adblock is on there is problem in upload result page showing square top of deletion section.

**Comment by Jaex** [comment]
I don't know what name to give to your uploader in destinations list?

**Comment by TechKat** [comment]
Are they faint-black squares? If so, yes I am aware of that and looking into it. vgy.me has been running for a good 2-3 years now, with many structure/UI/ideas changed to make it as simple and easy as possible.

The name of the uploader is just vgy.me. But thank you for the kind words about its simplicity!

**Comment by xzyxzy** [comment]
Seems to be broken.

2016-09-17 02:45:42.160 - Task failed. Filename: vgy.me_-_Simple_Image_Hosting_-_Google_Chrome_2016-09-17_02-45-38.png, Errors:
Message:
Невозможно соединиться с удаленным сервером _Unable to connect to remote server_

StackTrace:
   в System.Net.HttpWebRequest.GetRequestStream(TransportContext& context)
   в System.Net.HttpWebRequest.GetRequestStream()
   в ShareX.UploadersLib.Uploader.UploadData(Stream dataStream, String url, String fileName, String fileFormName, Dictionary`2 arguments, NameValueCollection headers, CookieCollection cookies, ResponseType responseType, HttpMethod method, String contentType, String metadata)

**Comment by TechKat** [comment]
@xzyxzy Hi, vgy.me admin here.

vgy.me's working fine: http://isup.me/vgy.me

Could you ping/traceroute vgy and make sure it exists in your DNS?

**Comment by xzyxzy** [comment]
It works in browser and traceroutes fine. It only doesn't works in ShareX.

**Comment by TechKat** [comment]
Are you on the latest version?

**Comment by xzyxzy** [comment]
Yes, I've tried on old version (don't remember which exactly, 11.2.0 probably) and then downloaded new version 11.2.1.

**Comment by Trost123** [comment]
@TechKat I'll assume @Jaex was right and you stopped supporting your service?
And now I also wasted some time registering (captcha didn't show at first) and setting everything up just to see an error.

1) Trying to acess https://vgy.me/upload
Gives me this:
![image](https://user-images.githubusercontent.com/21346282/62369423-42802080-b538-11e9-8304-f989cda231e0.png)
2) Trying to upload, using ShareX, gives me this:
image.png
![image](https://user-images.githubusercontent.com/21346282/62369461-5d529500-b538-11e9-9c2d-b66f4075a2b4.png)
:disappointed:

**Comment by TechKat** [comment]
@Trost123 I've just tested this and it's still working. I haven't stopped supporting ShareX, and I don't think @Jaex has stopped supporting vgy.me. Worth noting that the `/upload` path only supports POST requests. GET requests will throw a 404 Not Found.

**Comment by Trost123** [comment]
> @Trost123 I've just tested this and it's still working. I haven't stopped supporting ShareX, and I don't think @Jaex has stopped supporting vgy.me. Worth noting that the `/upload` path only supports POST requests. GET requests will throw a 404 Not Found.

Ok. Thanks for a swift response and sorry for wrong accusations.
I found out why it was happening: In ShareX settings, where I'm supposed to paste the user key, when I hit ctrl-v, only "***" appeared, despite user key supposed to be 10+ symbols. I noticed this, but dismissed it as a feature of ShareX to hide the password length. I even pasted the same clipboard in browser to double-check it.

Now, after getting your mail - I re-pasted it for the third time and it works. I wish the error was more informative though. Something like "Wrong user key" in response message.

Thanks again both of you for providing me with a great way of making direct-link screenshot.
Sadly Imgur and ImageShack  and TinyPic don't play fair and their "direct links" actually redirect to ad-ridden pages (on mobile only), where image is all messed up, not 100% zoom and quality.
Your service doesn't do this and I hope it stays that way.
